zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/BpmnProcessors.java

```diff
@@ -142,13 +145,14 @@ private static void addMessageStreamProcessors(
       final MutableProcessingState processingState,
       final Supplier<ScheduledTaskState> scheduledTaskState,
       final Writers writers,
-      final InstantSource clock) {
+      final InstantSource clock,
+      final TransientPendingSubscriptionState transientState) {
```

Comment:
🔧 There exist two different instances of `TransientPendingSubscriptionState`: one for message subscriptions and one for process message subscriptions. Let's be explicit about which is passed along here by continuing to use the same name: `transientProcessMessageSubscriptionState`.